libraries _PLUGIN145 ITHO FAN/Itho/IthoCC1101.cpp

```diff
@@ -986,12 +988,22 @@ bool IthoCC1101::checkID(const uint8_t *id)
 	return true;
 }
 
-String IthoCC1101::getLastIDstr() {
+String IthoCC1101::getLastIDstr(bool ashex) {
 	String str;
 	for (uint8_t i=0; i<8;i++) {
-		str += String(inIthoPacket.deviceId2[i], HEX);
-		if (i<7) str += "-";
+		if (ashex) str += String(inIthoPacket.deviceId2[i], HEX);
+		else str += String(inIthoPacket.deviceId2[i]);
+		if (i<7) str += ":";
```

Comment:
Please call right after declaring the String: `str.reserve(24);` or maybe even more if not using hex.

Reply:
Added. We call it now as non-hex in the plugin and expect 24 characters.